x/ccv/consumer/module.go

```diff
@@ -127,6 +127,9 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(consumertypes.ModuleName, 1, m.Migrate1to2); err != nil {
```

Comment:
Remove as v3.0.0 and v3.1.0 have consensus version 1 and they are both deprecated. The consumer module on >= v3.2.0 has consensus version 2.

Reply:
Let's leave the migrations as they are written now.

Removing the migrations would make it difficult to figure out what was happening and when. Having all migrations in front of you makes it somewhat easier.

This is from experience we got during provider v2 -> v3 migration which was a mess and some previously registered migrations were just deleted. Due to the deletion previous code versions needed to be cross-checked to make sure we're not dropping something.